src/bidders/prebid/adapters/appnexus.js

```diff
@@ -40,6 +36,10 @@ export class Appnexus extends BaseAdapter {
 			position = vertical && this.placements[vertical] ? vertical : 'other';
 		}
 
+		if (this.recPlacements && context.get('custom.rec')) {
```

Comment:
What about changing context (defined placements) in app repository instead of creating special logic for handling recovery case here?

Reply:
That's a great idea. I gonna handle this in app repository.